zipline/utils/run_algo.py

```diff
@@ -5,6 +5,10 @@
 import warnings
 
 import click
+
+from zipline.finance.blotter.blotter import Blotter
```

Comment:
we should import these things both from `zipline.finance.blotter` directly instead of going to the submodules. The reason to reexport the interface through the `__init__` is that consumers (like this file) don't need to know about the layout of the `blotter` directory.

Reply:
Done